### DISCUSSION TITLE
How to add tips

### DISCUSSION THREAD

**Discussion by MattNot** [securingsincity/react-ace#826] [Q&A]
Is there a way to add things like the "Alt-enter" function in JetBrains IDEs?
A contextual menu to suggest thing to the user like this:
![Screenshot](https://user-images.githubusercontent.com/29925647/79510792-532ce400-803e-11ea-805c-6006a72f1944.PNG)

**Comment by tageecc** [comment]
any updates?

**Comment by MattNot** [comment]
No, i've done on my own with a third-party lib saving on cursor move the line wich i am in a state (redux) and then using it in the context menu
